README.md

```
@@ -1,337 +1,67 @@
+---
 
-    dynamic value = 1;
-    value = false;
+## ⚙️ Como Executar
 
-## Palavras-chave
+Para rodar este projeto em sua máquina local, siga os passos abaixo:
 
-Se você quer que uma variável nunca seja alterada deve-se usar final ou const, em vez de usar var ou somente o tipo. Uma variável definida como final pode ter seu valor definido apenas uma vez, já uma variável const é uma constante de tempo de compilação. (Uma variável const é implicitamente final)
+1.  Clone este repositório:
+    ```sh
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)